### ISSUE TITLE
Error 105 received from application: Isolate must be runnable

### ISSUE THREAD

**Issue by HadiSDev** [flutter/flutter#34954]
## Steps to Reproduce

```
`dependencies:
  multi_page_form:
  transformer_page_view:
  flutter_walkthrough:
  encrypt: ^3.2.0
  flutter_secure_storage: ^3.2.1+1
  flutter_local_notifications: ^0.7.1+3
  shimmer: ^1.0.0
  flutter_crashlytics: ^1.0.0
  splashscreen: ^1.2.0
  permission_handler: ^3.1.0
  cached_network_image: ^0.8.0
  flutter_spinkit: ^3.1.0
  carousel_slider: ^1.3.0
  url_launcher: ^5.0.3
  dio: ^2.1.7
  firebase_auth: ^0.11.1+6
  rflutter_alert: ^1.0.2
  http: ^0.12.0+2
  flutter_auth_buttons: ^0.5.0
  flutter_facebook_login: ^2.0.1
  flutter_svg: ^0.10.2
  flutter:
    sdk: flutter
    
  dev_dependencies:
  json_serializable: ^3.0.0
  build_runner: ^1.0.0
  flutter_test:
    sdk: flutter

import 'package:flutter/material.dart';
import './StartPage.dart';
import './LoginPage.dart';
import './HomePage.dart';
import './Settings_page/SettingsPage.dart';
import './TabNavigator.dart';
//import 'SignUpUserPage.dart';

void main() => runApp(new HublyApp());

class HublyApp extends StatelessWidget {
  final routes = <String, WidgetBuilder>{
    StartPage.tag: (context) => StartPage(),
    LoginPage.tag: (context) => LoginPage(),
    HomePage.tag: (context) => HomePage(),
    //SignUpUserPage.tag: (context) => SignUpUserPage(),
    SettingsPage.tag: (context) => SettingsPage(),
    TabNavigator.tag: (context) => TabNavigator(),
  };

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      debugShowCheckedModeBanner: false,
      home: StartPage(),
      routes: routes,
    );
  }
}`
```

I don't know how the error occurred. The main function is correctly setup. I have tried multiple forum post but nothing worked

## Logs

        +65 ms] Scanned through 439 files in 60ms
[   +1 ms] Syncing files to device Android SDK built for x86...
[        ] Scanning asset files
[  +10 ms] Compiling dart to kernel with 0 updated files
[   +2 ms] <- recompile package:hubly_app/main.dart e44d721e-ab6d-49f9-934a-f36777b9c731
[   +4 ms] <- e44d721e-ab6d-49f9-934a-f36777b9c731
[   +4 ms] -> result a0765988-3fd4-4623-a190-7025460366b7
[  +28 ms] -> a0765988-3fd4-4623-a190-7025460366b7
[   +1 ms] -> a0765988-3fd4-4623-a190-7025460366b7 build\app.dill.incremental.dill 0
[   +1 ms] Updating files
[  +22 ms] DevFS: Sync finished
[   +7 ms] Syncing files to device Android SDK built for x86... (completed in 82ms)
[        ] Synced 0.0MB.
[   +3 ms] Sending to VM service: _flutter.setAssetBundlePath({isolateId: isolates/165584173, viewId: _flutterView/0xe9fde80c, assetDirectory:
/data/user/0/com.hubly.hublyapp/code_cache/hubly_appTOZPFA/hubly_app/build/flutter_assets})
[  +10 ms] Result: {type: Success, view: {type: FlutterView, id: _flutterView/0xe9fde80c, isolate: {type: @Isolate, fixedId: true, id: isolates/165584173, name: 
main.dart$main-165584173, number: 165584173}}}
[   +9 ms] Sending to VM service: _reloadSources({pause: false, rootLibUri:
file:///data/user/0/com.hubly.hublyapp/code_cache/hubly_appTOZPFA/hubly_app/lib/main.dart.incremental.dill, packagesUri:
file:///data/user/0/com.hubly.hublyapp/code_cache/hubly_appTOZPFA/hubly_app/.packages, isolateId: isolates/165584173})
[  +10 ms] Error 105 received from application: Isolate must be runnable
[   +2 ms] {request: {method: _reloadSources, params: {pause: false, rootLibUri:
file:///data/user/0/com.hubly.hublyapp/code_cache/hubly_appTOZPFA/hubly_app/lib/main.dart.incremental.dill, packagesUri:
file:///data/user/0/com.hubly.hublyapp/code_cache/hubly_appTOZPFA/hubly_app/.packages, isolateId: isolates/165584173}}, details: Isolate must be runnable before 
this request is made.}
Unhandled exception:
{code: 105, message: Isolate must be runnable, data: {request: {method: _reloadSources, params: {pause: false, rootLibUri: file:///data/user/0/com.hubly.hublyapp/code_cache/hubly_appTOZPFA/hubly_app/lib/main.dart.incremental.dill, packagesUri: file:///data/user/0/com.hubly.hublyapp/code_cache/hubly_appTOZPFA/hubly_app/.packages, isolateId: isolates/165584173}}, details: Isolate must be runnable before this request is made.}}
#0      _rootHandleUncaughtError.<anonymous closure> (dart:async/zone.dart:1112:29)
#1      _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#2      _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
#3      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:115:13)
#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:172:5)

```
```

 info - Unused import: 'dart:async' - lib\Client.dart:1:8 - unused_import
   info - Unused import: 'dart:convert' - lib\Client.dart:2:8 - unused_import
   info - Unused import: 'package:http/http.dart' - lib\HomePage.dart:3:8 - unused_import
   info - The value of the field '_auth' isn't used - lib\LoginPage.dart:18:16 - unused_field
   info - Unused import: 'package:json_serializable/builder.dart' - lib\Models\UserModels\UserModels.dart:1:8 - unused_import
   info - The function '_$CreateUserDTOFromJson' isn't used - lib\Models\UserModels\UserModels.g.dart:109:15 - unused_element
   info - Name non-constant identifiers using lowerCamelCase - lib\Models\Users_repository\UserRepository.dart:16:23 - non_constant_identifier_names
   info - Name non-constant identifiers using lowerCamelCase - lib\Settings_page\Legal_settings\legal_settings.dart:5:7 - non_constant_identifier_names
   info - Name non-constant identifiers using lowerCamelCase - lib\Settings_page\Logout_settings\logout_settings.dart:5:7 - non_constant_identifier_names        
   info - Name non-constant identifiers using lowerCamelCase - lib\Settings_page\Notification_settings\notification_settings.dart:5:7 -
          non_constant_identifier_names
   info - Name non-constant identifiers using lowerCamelCase - lib\Settings_page\Profile_settings\profile_settings.dart:5:7 - non_constant_identifier_names      
   info - Unused import: 'colors.dart' - lib\Settings_page\SettingsPage.dart:4:8 - unused_import
   info - Unused import: 'header.dart' - lib\Settings_page\SettingsPage.dart:6:8 - unused_import
   info - Unused import: 'item.dart' - lib\Settings_page\SettingsPage.dart:7:8 - unused_import
   info - Name non-constant identifiers using lowerCamelCase - lib\Settings_page\Support_settings\support_settings.dart:5:7 - non_constant_identifier_names      
   info - Unused import: 'package:flutter/material.dart' - lib\Settings_page\deleteProfile_settings\deleteprofile_settings.dart:2:8 - unused_import
   info - Unused import: 'package:flutter/cupertino.dart' - lib\Settings_page\deleteProfile_settings\deleteprofile_settings.dart:3:8 - unused_import
   info - Name non-constant identifiers using lowerCamelCase - lib\Settings_page\deleteProfile_settings\deleteprofile_settings.dart:5:7 -
          non_constant_identifier_names
   info - Unused import: 'group.dart' - lib\Settings_page\group.dart:4:8 - unused_import
   info - Unused import: 'header.dart' - lib\Settings_page\group.dart:5:8 - unused_import
   info - Unused import: 'package:multi_page_form/multi_page_form.dart' - lib\SignUpUserPage.dart:4:8 - unused_import

```
```

[√] Flutter (Channel stable, v1.5.4-hotfix.2, on Microsoft Windows [Version 10.0.18362.175], locale en-DK)
    • Flutter version 1.5.4-hotfix.2 at G:\flutter
    • Framework revision 7a4c33425d (8 weeks ago), 2019-04-29 11:05:24 -0700
    • Engine revision 52c7a1e849
    • Dart version 2.3.0 (build 2.3.0-dev.0.5 a1668566e5)

 
[√] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at G:\Android-sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.0
    • ANDROID_HOME = G:\Android-sdk
    • ANDROID_SDK_ROOT = C:\Program Files (x86)\Android\android-sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[√] Android Studio (version 3.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 33.4.1
    • Dart plugin version 182.5215
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[!] IntelliJ IDEA Ultimate Edition (version 2018.1)
    • IntelliJ at G:\Intellij\IntelliJ IDEA 2018.1
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[!] IntelliJ IDEA Ultimate Edition (version 2018.3)
    • IntelliJ at G:\Intellij\IntelliJ IDEA 2018.3.1
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[√] VS Code (version 1.35.1)
    • VS Code at C:\Users\Hadi\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.1.0

[√] VS Code, 64-bit edition (version 1.26.0)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.1.0

[√] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)

! Doctor found issues in 2 categories.

```
```
![image](https://user-images.githubusercontent.com/10367624/59977251-4599ff80-95cf-11e9-9b19-414f03a0c036.png)

A white screen appears with the error:
E/flutter ( 7023): [ERROR:flutter/shell/common/shell.cc(184)] Dart Error: error: import of dart:mirrors with --enable-mirrors=false
E/flutter ( 7023): [ERROR:flutter/shell/common/engine.cc(175)] Could not prepare to run the isolate.
E/flutter ( 7023): [ERROR:flutter/shell/common/engine.cc(122)] Engine not prepare and launch isolate.
E/flutter ( 7023): [ERROR:flutter/shell/platform/android/android_shell_holder.cc(167)] Could not launch engine in configuration.
D/EGL_emulation( 7023): eglMakeCurrent: 0xf17c6840: ver 3 0 (tinfo 0xf17b19f0)
Syncing files to device Android SDK built for x86...
 2,906ms (!)

�🔥  To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
An Observatory debugger and profiler on Android SDK built for x86 is available at: http://127.0.0.1:53285/jqRuHHossTY=/
For a more detailed help message, press "h". To detach, press "d"; to quit, press "q".

Initializing hot reload...                                             |Error 105 received from application: Isolate must be runnable
Unhandled exception:
{code: 105, message: Isolate must be runnable, data: {request: {method: _reloadSources, params: {pause: false, rootLibUri: file:///data/user/0/com.hubly.hublyapp/code_cache/hubly_appDUHZMK/hubly_app/lib/main.dart.incremental.dill, packagesUri: file:///data/user/0/com.hubly.hublyapp/code_cache/hubly_appDUHZMK/hubly_app/.packages, isolateId: isolates/972266956}}, details: Isolate must be runnable before this request is made.}}
#0      _rootHandleUncaughtError.<anonymous closure> (dart:async/zone.dart:1112:29)
#1      _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#2      _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
#3      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:115:13)
#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:172:5)

**Comment by HadiSDev** [comment]
I have also tried to just return a simple MaterialApp in main.dart instead:
`    return MaterialApp(
      color: Colors.blueAccent,
    );`
But that does not work either

**Comment by wangjue666** [comment]
i have had this problem too.
Did you solve it now?

**Comment by zekihocanet** [comment]
i have the same issue:


> E/flutter (11264): [ERROR:flutter/shell/common/engine.cc(175)] Could not prepare to run the isolate.
> E/flutter (11264): [ERROR:flutter/shell/common/engine.cc(122)] Engine not prepare and launch isolate.
> E/flutter (11264): [ERROR:flutter/shell/platform/android/android_shell_holder.cc(167)] Could not launch engine in configuration.

@HadiSDev did you solve this problem?

**Comment by HadiSDev** [comment]
Yes I did. Can I see your dependencies also in dev.
@zekihocanet

**Comment by HadiSDev** [comment]
@wangjue666 probably a dependency problem. Can I see your pub yaml file

**Comment by lucien-theron** [comment]
Same issue, my pubspec.yaml:
`name: ng_sales_rep_app
description: A new Flutter project.

# The following defines the version and build number for your application.
# A version number is three numbers separated by dots, like 1.2.43
# followed by an optional build number separated by a +.
# Both the version and the builder number may be overridden in flutter
# build by specifying --build-name and --build-number, respectively.
# Read more about versioning at semver.org.
version: 0.3.11+36

environment:
  sdk: ">=2.1.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter

  charts_flutter: ^0.6.0
  dio: ^2.1.13
  url_launcher: ^5.0.5
  flutter_redux: ^0.5.3
  redux_persist: ^0.8.2
  redux_persist_flutter: ^0.8.1
  path_provider: ^0.5.0+1 # Plugin redux_persist_flutter 0.8.1 has dependency on this version
  flutter_udid: ^1.0.0
  device_info: ^0.4.0+2
  event_bus: ^1.1.0
  datetime_picker_formfield: ^0.2.0
  google_maps_flutter: ^0.5.19+2
  android_alarm_manager: # Waiting on https://github.com/flutter/flutter/issues/34110 to stable
    git:
      url: https://github.com/flutter/plugins
      path: packages/android_alarm_manager
  flutter_email_sender: ^2.0.3
  package_info: ^0.4.0+4
  webview_flutter: ^0.3.9+1
  flutter_local_notifications: ^0.6.0 # Needs a gradle update before updating > 0.6.0
  connectivity: ^0.4.3+4
  uuid: ^2.0.2
  file_picker: ^1.3.7
  open_file: ^2.0.3
  geolocator: ^4.0.3

  # The following adds the Cupertino Icons font to your application.
  # Use with the CupertinoIcons class for iOS style icons.
  cupertino_icons: ^0.1.2

dev_dependencies:
  flutter_test:
    sdk: flutter

  flutter_launcher_icons: ^0.7.0

flutter_icons:
  android: true
  ios: true
  image_path: "assets/logo.png"

# For information on the generic Dart part of this file, see the
# following page: https://www.dartlang.org/tools/pub/pubspec

# The following section is specific to Flutter.
flutter:

  # The following line ensures that the Material Icons font is
  # included with your application, so that you can use the icons in
  # the material Icons class.
  uses-material-design: true

  # To add assets to your application, add an assets section, like this:
  assets:
    - assets/NG_Syspro_Logo_Menu.png
    - assets/bg-005.jpg
    - assets/bg-045.jpg
    - assets/map_markers/celestemarker.png
    - assets/map_markers/darkbluemarker.png
    - assets/map_markers/darkredmarker.png
    - assets/map_markers/electricyellowmarker.png
    - assets/map_markers/greenmarker.png
    - assets/map_markers/orangemarker.png
    - assets/map_markers/pinkmarker.png
    - assets/map_markers/shamrockmarker.png

  # An image asset can refer to one or more resolution-specific "variants", see
  # https://flutter.io/assets-and-images/#resolution-aware.

  # For details regarding adding assets from package dependencies, see
  # https://flutter.io/assets-and-images/#from-packages

  # To add custom fonts to your application, add a fonts section here,
  # in this "flutter" section. Each entry in this list should have a
  # "family" key with the font family name, and a "fonts" key with a
  # list giving the asset and other descriptors for the font. For
  # example:
  # fonts:
  #   - family: Schyler
  #     fonts:
  #       - asset: fonts/Schyler-Regular.ttf
  #       - asset: fonts/Schyler-Italic.ttf
  #         style: italic
  #   - family: Trajan Pro
  #     fonts:
  #       - asset: fonts/TrajanPro.ttf
  #       - asset: fonts/TrajanPro_Bold.ttf
  #         weight: 700
  #
  # For details regarding fonts from package dependencies,
  # see https://flutter.io/custom-fonts/#from-packages
`

**Comment by lucien-theron** [comment]
Any updates on this?

**Comment by jonahwilliams** [comment]
We think this is fixed in the latest stable, @lucien-theron do you still get this error in 1.9.1 ?

**Comment by lucien-theron** [comment]
Hi @jonahwilliams sorry for delay. Just tested and I am still getting the error:

![image](https://user-images.githubusercontent.com/7767248/65677746-710b0b00-e052-11e9-8cf0-d98919d5d033.png)

![image](https://user-images.githubusercontent.com/7767248/65677808-8b44e900-e052-11e9-86e0-d9deef3848f3.png)

**Comment by theshivamlko** [comment]
I got this error on Hot Reload/Restart after I upgrade master for Flutter Desktop.
The error is in all existing mobile projects on stable also.
I mentioned here #42834 in detail

**Comment by HadiSDev** [comment]
I am fairly sure it has something to do with packages/plugins not working together. The way I fixed it was to remove packages one by one and each time see if error is still thrown.

**Comment by emiliodallatorre** [comment]
This error, I noticed, started happening the day next I upgraded Android Studio to 3.5.1.

**Comment by kliyer-ai** [comment]
@HadiSDev Can you let me know what package or combination of packages you had to remove? I am trying to fix this error for my application and would be nice to know what caused the issue for you.

**Comment by HadiSDev** [comment]
The two packages which were conflicting were json_serializable and json_annotation. The reason being that json_serializable already includes json_annotation, so including that separately messed up the package version.

**Comment by kliyer-ai** [comment]
Thanks for answering. I finally found my issue. It was related to Firebase messaging. 
I was calling
```
_firebaseMessaging.configure(
      onMessage: onMessage(context),
      onBackgroundMessage: myBackgroundMessageHandler,
      onLaunch: onLaunch(context),
      onResume: onResume(context),
 );
```
in the `build` method of my top most widgets in the widget tree. Removing the background message handler and moving it to the initState method of a stateful widget solved the problem for now.

**Comment by bayuramadeza** [comment]
> Thanks for answering. I finally found my issue. It was related to Firebase messaging.
> I was calling
> 
> ```
> _firebaseMessaging.configure(
>       onMessage: onMessage(context),
>       onBackgroundMessage: myBackgroundMessageHandler,
>       onLaunch: onLaunch(context),
>       onResume: onResume(context),
>  );
> ```
> 
> in the `build` method of my top most widgets in the widget tree. Removing the background message handler and moving it to the initState method of a stateful widget solved the problem for now.

May i know how your  code after moving background to iniut state?

**Comment by mrbech** [comment]
I am also getting this error when hot reloading with Firebase. It only happens when setting `onBackgroundMessage` on Android.

**Comment by ggutenberg** [comment]
> > Thanks for answering. I finally found my issue. It was related to Firebase messaging.
> > I was calling
> > ```
> > _firebaseMessaging.configure(
> >       onMessage: onMessage(context),
> >       onBackgroundMessage: myBackgroundMessageHandler,
> >       onLaunch: onLaunch(context),
> >       onResume: onResume(context),
> >  );
> > ```
> > 
> > 
> > in the `build` method of my top most widgets in the widget tree. Removing the background message handler and moving it to the initState method of a stateful widget solved the problem for now.
> 
> May i know how your code after moving background to iniut state?

This is working for me:

```flutter
class _LandingPage extends State<LandingPage> {
  final FirebaseMessaging _firebaseMessaging = FirebaseMessaging();

  static Future<dynamic> onMessage(
      BuildContext context, Map<String, dynamic> message) async {
    Messaging.onMessage(message);  // external lib
    alert(message.toString(), context);  // external lib
  }

  static Future<dynamic> onResume(
      BuildContext context, Map<String, dynamic> message) async {
    Messaging.onResume(message);  // external lib
    alert(message.toString(), context);  // external lib
  }

  static Future<dynamic> onLaunch(
      BuildContext context, Map<String, dynamic> message) async {
    Messaging.onLaunch(message);  // external lib
    alert(message.toString(), context);  // external lib
  }

  @override
  void initState() {
    _firebaseMessaging.requestNotificationPermissions(
        const IosNotificationSettings(sound: true, badge: true, alert: true));
    _firebaseMessaging.configure(
      onBackgroundMessage:
          Messaging.myBackgroundMessageHandler,  // external lib
    );
    _firebaseMessaging.subscribeToTopic('all');
  }

  @override
  Widget build(BuildContext context) {
    _firebaseMessaging.configure(
      onMessage: (message) async => onMessage(context, message),
      onLaunch: (message) async => onLaunch(context, message),
      onResume: (message) async => onResume(context, message),
    );

    ...
```

**Comment by jonahwilliams** [comment]
There are a couple different issues being discussed here. The original bug was a configuration issue which has long since been fixed. For the firebase messaging problem, I would recommend that someone who experienced and fixed the bug file a new issue detailing the problem and what you did. we can use that to update documention

**Comment by bayuramadeza** [comment]
> > > Thanks for answering. I finally found my issue. It was related to Firebase messaging.
> > > I was calling
> > > ```
> > > _firebaseMessaging.configure(
> > >       onMessage: onMessage(context),
> > >       onBackgroundMessage: myBackgroundMessageHandler,
> > >       onLaunch: onLaunch(context),
> > >       onResume: onResume(context),
> > >  );
> > > ```
> > > 
> > > 
> > > in the `build` method of my top most widgets in the widget tree. Removing the background message handler and moving it to the initState method of a stateful widget solved the problem for now.
> > 
> > 
> > May i know how your code after moving background to iniut state?
> 
> This is working for me:
> 
> ```dart
> class _LandingPage extends State<LandingPage> {
>   final FirebaseMessaging _firebaseMessaging = FirebaseMessaging();
> 
>   static Future<dynamic> onMessage(
>       BuildContext context, Map<String, dynamic> message) async {
>     Messaging.onMessage(message);  // external lib
>     alert(message.toString(), context);  // external lib
>   }
> 
>   static Future<dynamic> onResume(
>       BuildContext context, Map<String, dynamic> message) async {
>     Messaging.onResume(message);  // external lib
>     alert(message.toString(), context);  // external lib
>   }
> 
>   static Future<dynamic> onLaunch(
>       BuildContext context, Map<String, dynamic> message) async {
>     Messaging.onLaunch(message);  // external lib
>     alert(message.toString(), context);  // external lib
>   }
> 
>   @override
>   void initState() {
>     _firebaseMessaging.requestNotificationPermissions(
>         const IosNotificationSettings(sound: true, badge: true, alert: true));
>     _firebaseMessaging.configure(
>       onBackgroundMessage:
>           Messaging.myBackgroundMessageHandler,  // external lib
>     );
>     _firebaseMessaging.subscribeToTopic('all');
>   }
> 
>   @override
>   Widget build(BuildContext context) {
>     _firebaseMessaging.configure(
>       onMessage: (message) async => onMessage(context, message),
>       onLaunch: (message) async => onLaunch(context, message),
>       onResume: (message) async => onResume(context, message),
>     );
> 
>     ...
> ```

i want to know, why did you configure fcm in buildwidget instead of in initstate?

**Comment by hmawla** [comment]
Any updates on this issue?

**Comment by yxwandroid** [comment]
我运行在ios 上显示白屏

I'm running on ios with a white screen

[✓] Flutter (Channel stable, v1.12.13+hotfix.8, on Mac OS X 10.15.4 19E266, locale zh-Hans-CN)
    • Flutter version 1.12.13+hotfix.8 at /Users/wilson/aochuang/FlutterSDK
    • Framework revision 0b8abb4724 (7 weeks ago), 2020-02-11 11:44:36 -0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0

[!] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/wilson/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/wilson/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4, Build version 11E146
    • CocoaPods version 1.9.0

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] IntelliJ IDEA Ultimate Edition (version 2019.3.3)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 44.0.3
    • Dart plugin version 193.6494.35
